openeo_udp/biopar/biopar.json

```diff
@@ -59,8 +59,8 @@
             }
         },
         {
-            "description": "This geometries defines the bounds of the resulting image. Pixels outside the polygon are set to nodata.",
-            "name": "geometries",
+            "description": "This spatial_extent defines the bounds of the resulting image. Pixels outside the polygon are set to nodata.",
```

Comment:
This is a bit confusing:  "defines the bounds" seems to suggest that `spatial_extent` is a bounding box construct (like in the prototypical `load_collection` usage), but the type  of this parameter is "geojson", so a bbox isn't even valid